modules/ROOT/pages/dataweave-cookbook-java-methods.adoc

```diff
@@ -116,17 +131,41 @@ output application/json
 }
 ----
 
-This should output the following:
+To add this DataWeave example to your Studio project:
 
+. Returning to `my-app.xml` in *src/main/mule*, drag a Transform Message
+component into the project's canvas, and click to open the component.
```

Comment:
maybe change it to "and click to open it."

Reply:
@leansh done